source/test/diff.ts

```diff
@@ -0,0 +1,58 @@
+import {verifyWithDiff} from './fixtures/utils';
+import execa, {ExecaError} from 'execa';
+import path from 'path';
+import test from 'ava';
+import tsd from '..';
+
+test('diff', async t => {
+	const diagnostics = await tsd({cwd: path.join(__dirname, 'fixtures/diff')});
+
+	verifyWithDiff(t, diagnostics, [
+		[8, 0, 'error', 'Parameter type `{ life?: number | undefined; }` is declared too wide for argument type `{ life: number; }`.', {
+			expected: '{ life?: number | undefined; }',
+			received: '{ life: number; }',
+		}],
+		[9, 0, 'error', 'Parameter type `FooFunction` is not identical to argument type `() => number`.', {
+			expected: '(x?: string | undefined) => number',
+			received: '() => number',
+		}],
+		[10, 0, 'error', 'Parameter type `FooType` is declared too wide for argument type `Required<FooType>`.', {
+			expected: '{ foo?: \'foo\' | undefined; }',
+			received: '{ foo: \'foo\'; }',
+		}],
+		[11, 0, 'error', 'Parameter type `Partial<FooInterface>` is declared too wide for argument type `Required<FooInterface>`.', {
+			expected: '{ [x: string]: unknown; readonly protected?: boolean | undefined; fooType?: FooType | undefined; id?: \'foo-interface\' | undefined; }',
+			received: '{ [x: string]: unknown; readonly protected: boolean; fooType: FooType; id: \'foo-interface\'; }',
+		}]
+	]);
+});
+
+test('diff cli', async t => {
+	const file = path.join(__dirname, 'fixtures/diff');
+
+	const {exitCode, stderr} = await t.throwsAsync<ExecaError>(execa('dist/cli.js', [file, '--showDiff']));
```

Comment:
```suggestion
	const {exitCode, stderr} = await t.throwsAsync<ExecaError>(execa('dist/cli.js', [file, '--show-diff']));
```

Reply:
`meow` accepts both syntaxes, but if you prefer this I'm fine with it ;)